src/plugins/vis_type_vega/public/vega_view/vega_map_view.js

```diff
@@ -144,7 +144,8 @@ export class VegaMapView extends VegaBaseView {
         bindingsContainer: this._$controls.get(0),
         delayRepaint: mapConfig.delayRepaint,
         viewConfig: this._vegaViewConfig,
-        viewOptions: this._vegaViewOptions,
+        parseConfig: null,
+        parseOptions: this._vegaViewOptions,
```

Comment:
this was a minor version change?

Reply:
Should have been a patch :)

Reply:
without knowing any context about these properties - is it correct to still set `viewConfig` and `parseConfig` to null? That seems... weird.

Reply:
This to make sure it matches how we use `vega.parse`:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/bebbcca30d4b3f43c800eb5360681d2072d0ba7c/src/plugins/vis_type_vega/public/vega_view/vega_view.js#L40